src/Whoops/Resources/js/whoops.base.js

```diff
@@ -94,20 +104,34 @@ Zepto(function($) {
     return actionMsg;
   }
 
+  function scrollIntoView($node, $parent) {
```

Comment:
Although the commit message is supposed to explain something, I still don't fully understand what this does differently. Could you explain?

- [ ] Clarify custom `scrollIntoView`